packages/a11y/src/index.ts

```diff
@@ -0,0 +1,75 @@
+/* tslint:disable:no-console */
```

Comment:
It would be nice to implement a `log` method in utils so that we only have to disable console in one place. @AviVahl showed me this the other day